Meta/Lagom/CMakeLists.txt

```diff
@@ -95,31 +95,6 @@ if (EMSCRIPTEN)
     add_cxx_link_options(--emrun "SHELL:-s ALLOW_MEMORY_GROWTH")
 endif()
 
-if (ENABLE_ADDRESS_SANITIZER)
-    add_cxx_compile_options(-fsanitize=address -fno-omit-frame-pointer)
-    add_cxx_link_options(-fsanitize=address)
-    add_swift_compile_options(-sanitize=address)
-    add_swift_link_options(-sanitize=address)
-endif()
-
-if (ENABLE_MEMORY_SANITIZER)
-    add_cxx_compile_options(-fsanitize=memory -fsanitize-memory-track-origins -fno-omit-frame-pointer)
-    add_cxx_link_options(-fsanitize=memory -fsanitize-memory-track-origins)
-endif()
-
-if (ENABLE_UNDEFINED_SANITIZER AND (APPLE OR NOT ENABLE_SWIFT))
```

Comment:
you can't just remove this. Move it to one of the .cmake files and remove it from both. We still use Meta/Lagom as a source dir in a few esoteric places.